packages/scss/src/components/card/vars.scss

```diff
@@ -1,5 +1,5 @@
 @mixin vars {
-	--components-card-content-padding: var(--spacings-M);
+	--components-card-content-padding: .75rem var(--spacings-S);
 	--components-card-footer-padding: 0.33rem var(--spacings-M);
```

Comment:
0.33 rem?

Reply:
Old code for a mod not mentioned on Figma.
Fixed anyway 🙂